tests/NovaTestTraitTest.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+namespace RonasIT\Support\Tests;
+
+use Illuminate\Database\Query\Builder;
+use Illuminate\Database\Schema\MySqlBuilder;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\DB;
+use Mockery;
+use RonasIT\Support\Tests\Support\Traits\MockTrait;
+use RonasIT\Support\Traits\FixturesTrait;
+use RonasIT\Support\Traits\NovaTestTrait;
+
+class NovaTestTraitTest extends HelpersTestCase
+{
+    use FixturesTrait, MockTrait, NovaTestTrait;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        self::$tables = [];
+
+        putenv('FAIL_EXPORT_JSON=false');
+    }
+
+    public function testActingAsNovaUser()
+    {
+        $userId = 1;
+        $this->novaActingAs($userId);
+        $session = $this->app['session']->getDrivers()['array'];
+
+        $loginSession = array_filter($session->all(), function ($key) {
+            return str_starts_with($key, 'login_session_');
+        }, ARRAY_FILTER_USE_KEY);
+
+        $this->assertNotEmpty($loginSession);
+        $this->assertEquals('laravel_session', $session->getName());
+        $this->assertTrue(array_values($loginSession)[0] === $userId);
+    }
+
+    public function testCacheJsonFields()
+    {
+        Config::set('database.default', 'mysql');
+
+        $mock = Mockery::mock('overload:' . MysqlBuilder::class);
+        $mock->shouldReceive('getColumnListing')
+            ->andReturn(['id', 'json_column'])
+
+            ->shouldReceive('getColumnType')
+            ->with('users', 'id')
+            ->andReturn('int')
+
+            ->shouldReceive('getColumnType')
+            ->with('users', 'json_column')
+            ->andReturn('json');
```

Comment:
let's move it to the separate test trait `mockGetColumnLis(['id' => 'int', 'json_column' => 'json'])`